samples/bootloader/src/main.c

```diff
@@ -123,7 +128,13 @@ static void validate_and_boot(const struct fw_info *fw_info, uint16_t slot)
 
 int main(void)
 {
-	int err = fprotect_area(PM_B0_ADDRESS, PM_B0_SIZE);
+	int err = 0;
+
+	if (IS_ENABLED(CONFIG_FPROTECT)) {
+		err = fprotect_area(PM_B0_ADDRESS, PM_B0_SIZE);
+	} else {
+		printk("Error: Failed to protect bootloader due to fprotect is not enabled.\n\r");
```

Comment:
Should this be error? The one below is "Fprotect disabled. No protection applied"